python/tests/io/test_pyarrow.py

```diff
@@ -139,7 +138,7 @@ def test_raise_on_opening_a_local_file_not_found():
 
 
 def test_raise_on_opening_a_local_file_no_permission():
-    """Test that a PyArrowFile raises appropriately when opening a local file without permission"""
+    """Test that a PyArrowFile raises appropriately when creating a local file without permission"""
```

Comment:
I think this should be kept to "opening" right? The test is creating a temp directory with locked down permissions, and then asserting that the right exception is raised when trying to open a file in that directory.